lib/src/builder/codegen.dart

```diff
@@ -35,9 +35,10 @@ class ImplGenerator {
   void generate(BoilerplateDeclaration declaration) {
     switch (declaration.type) {
       case DeclarationType.propsMapViewOrFunctionComponentDeclaration:
-        _generatePropsMapViewOrFunctionComponent(declaration);
-        break;
-      case DeclarationType.genericFunctionComponentDeclaration:
+        // Generated code only needed for props config.
+        if ((declaration as PropsMapViewOrFunctionComponentDeclaration).props != null) {
```

Comment:
We should be able to get rid of this check now that we're not yielding functions that don't need generation, right?